package.json

```diff
@@ -472,6 +472,16 @@
           "default": [],
           "markdownDescription": "Extra arguments to pass to ocamllsp."
         },
+        "ocaml.server.codelens": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable/Disable codelens"
+        },
```

Comment:
I'm not a fan of having this option in addition to `editor.codeLens`, but I'm not sure if it's ok to rely only on `editor.codeLens`

Reply:
When the `editor.codelens` option is disabled the client (vscode) continues making requests to the server, but does not render.

In the past I have observed this behavior in the server trace log.